src/factory/online.js

```diff
@@ -42,10 +42,14 @@ function SplitFactoryOnline(context, readyTrackers, mainClientMetricCollectors)
       break;
     }
     case CONSUMER_MODE:
+      setTimeout(() => {
```

Comment:
I'll add the comment to the code as well, but basically I need to allow for all the synchronic statements to run, in order to return the client before the SDK is considered ready. Couldn't use setImmediate since I need it to run on the browser.

Reply:
Addressed.